src/text-transform/transformers.ts

```diff
@@ -0,0 +1,79 @@
+import _ from 'lodash';
+
+import {ConstructorBlock} from '../models/constructor';
+import {Lang} from '../utils/configure';
+
+import {Transformer} from './common';
+import {BlocksConfig, config} from './config';
+
+export type ContentTransformerProps = {
+    content: {
+        blocks?: ConstructorBlock[];
+    };
+    options: {
+        lang: Lang;
+        customConfig?: {};
+    };
+};
+
+function transformBlocks(blocks: ConstructorBlock[], lang: Lang, customConfig = {}) {
+    const fullConfig = {...config, ...customConfig};
+
+    const clonedBlocks = _.cloneDeep(blocks);
```

Comment:
not necessary now but it would be better to rework this with no `cloneDeep`, but all in immutable style, it allows to avoid side effects errors and mixin both immutable and non immutable styles